style.css

```diff
@@ -412,7 +412,12 @@ h4{
   margin-left: 5px;
 }
 .faq-answer{
-  padding: 10px 0px;
+  padding: 18px 0px;
+  font-size: 16px;
+  text-align:left;
+}
+.faq-answer > a{
+	color: #aaa;
```

Comment:
The gray looks a bit off, maybe use the same color or something like `#9cf` for the link?
Before:
![image](https://user-images.githubusercontent.com/54484616/132104503-794c0a66-ca05-40d7-afc6-3e4cec0076d1.png)
After:
![image](https://user-images.githubusercontent.com/54484616/132104484-18eeeb2d-e54c-4b9b-bee2-bd79627e7316.png)